idaes/models/flowsheets/demo_flowsheet.py

```diff
@@ -56,397 +56,40 @@ def build_flowsheet():
 
 def set_scaling(m):
     """Set scaling for demo flowsheet"""
-
-    iscale.set_scaling_factor(m.fs.M01.inlet_1_state[0].flow_mol_phase, 1e6)
+    params = m.fs.BT_props
```

Comment:
Are all these changes necessary, especially seeing as you are xfailling the tests?

Reply:
Originally, I was hoping that scaling would fix the problems, but the problems are deeper than that. However, I thought that writing all the scaling in a more concise way (as well as removing some bad manual scaling that's now taken care of in the BTX properties) was a change worth keeping.